src/components/pages/Categories/Categories.vue

```diff
@@ -39,10 +51,32 @@ export default {
     errorMessage() {
       return this.$store.state.categories.errorMessage;
     },
+    doneMessage() {
+      return this.$store.state.categories.doneMessage;
+    },
   },
   created() {
     this.$store.dispatch('categories/getAllCategories');
   },
+  methods: {
+    clearMessage() {
+      this.$store.dispatch('categories/clearMessage');
+    },
+    resetMessage() {
+      this.$store.dispatch('categories/resetMessage');
+    },
+    updateValue(event) {
+      const { target } = event;
+      this.$store.dispatch('categories/updateValue', target.value);
+    },
+    handleSubmit() {
+      const categoryName = this.category;
+      this.$store.dispatch('categories/createCategories', categoryName).then(() => {
```

Comment:
[nits]

こちらも一時変数ですね。

```suggestion
      this.$store.dispatch('categories/createCategories', this.categoryName).then(() => {
```

Reply:
一時変数を削除しました。